docs/content/users/extend/in-container-configuration.md

```diff
@@ -1,28 +1,31 @@
-# In-Container home directory and shell configuration
+# In-Container Home Directory and Shell Configuration
 
-Custom shell configuration (bash or your preferred shell), your usual git configuration, a composer auth.json and more can be achieved within your containers.  Place all your dotfiles in your global`~/.ddev/homeadditions` or your project's `.ddev/homeadditions` directory and DDEV will use these in your project's web containers.  (Note that there is also a hidden/transient `.ddev/.homeadditions`; this is used for processing global homeadditions and should be ignored.)
+Custom shell configuration (Bash or your preferred shell), your usual Git configuration, a Composer `auth.json` and more can be achieved within your containers.
 
-On `ddev start`, ddev attempts to create a user inside the web and db containers with the same name and use id as the one you have on the host.
+Place all your dotfiles in your global`~/.ddev/homeadditions` or your project’s `.ddev/homeadditions` directory and DDEV will use these in your project’s `web` containers.
 
-DDEV looks for the `homeadditions` directory either in `~/.ddev/homeadditions` (the global .ddev directory) or the `.ddev/homeadditions` directory of a particular project, and will copy their contents recursively into the in-container home directory during `ddev start`. (Note that project homeadditions contents override the global homeadditions.)
+!!!tip "Ignore `.ddev/.homeadditions`!"
+    A hidden/transient `.ddev/.homeadditions`—emphasis on the leading `.`—is used for processing global `homeadditions` and should be ignored.
```

Comment:
Whoops! I had pulled that out of [the existing description just above](https://github.com/drud/ddev/blob/master/docs/content/users/extend/in-container-configuration.md?plain=1#L3) and specifically called out the `.` because it was easy to miss. 

The page goes on to describe `.ddev/homeadditions` (without the leading period)—is _that_ accurate? 🤔

Reply:
In both the project and global, it's `.ddev/homeadditions`

People should *not* look in `.ddev/.homeadditions` Maybe it's a bad idea to even mention that.

Reply:
Your call! I have no idea if people commonly confuse `.ddev/homeadditions` and `.ddev/.homeadditions`, I was just trying to scoot that mention into an aside to _reading_ about such similar-looking things would be more clear.

Reply:
I’m going to run with this in its _revised_ format for now, since it was in the original text, and if we still want to delete it then cool.